Dockerfile

```diff
@@ -12,5 +12,11 @@ RUN pip3 install -q -r /tmp/requirements.txt
 
 COPY deploy/mail/ssmtp.conf /etc/ssmtp/ssmtp.conf
 
+# Add crontask file in the cron directory
+ADD crontask /etc/cron.d/cron_caching
```

Comment:
use COPY instead of ADD and improve filename, maybe /etc/cron.daily/catalog_nightly_tasks or something